### ISSUE TITLE
Ask regarding the dataset

### ISSUE THREAD

**Issue by sonata13** [IIGROUP/AHIQ#4]
Hi, thanks for your hard work and congratulations for your 1st place in the CPVR competition,
I am curious on the implementation of your model. and currently, I have tried to deploy your model. Here, I have question:

1. Where do you get the PIPAL dataset? is it from https://drive.google.com/drive/folders/1G4fLeDcq6uQQmYdkjYUHhzyel4Pz81p- ?
is that true? If that so, from that link I saw only training data without validation and test.

or maybe could you share the dataset?

Thank you! really appreciate it.

**Comment by ch-andrei** [comment]
PIPAL dataset can be found here. 
https://competitions.codalab.org/competitions/28050
Make an account and you can download the data

**Comment by ch-andrei** [comment]
https://codalab.lisn.upsaclay.fr/competitions/1567
for the 2022 competition page

**Comment by ch-andrei** [comment]
https://codalab.lisn.upsaclay.fr/competitions/1567
It's part of the NTIRE22 competition. Make an account and you can download train/val/test data. Only the training set has publicly available quality labels.
